pkg/client/client.go

```diff
@@ -73,19 +73,23 @@ func UpdateAnnotation(obj *unstructured.Unstructured, oldID, newID string) (bool
 	val, found := annotations[key]
 	if !found || val == oldID {
 		annotations[key] = newID
+		return true, UpdateAnnotation(obj, annotations)
+	}
+	return false, nil
+}
+
+func UpdateAnnotation(obj *unstructured.Unstructured, annotations map[string]string) error {
```

Comment:
Since this is exported, we should probably have comments to describe its use.

Reply:
done